common/helpers/front_end.py

```diff
@@ -50,6 +50,22 @@ def get_page_path_parameters(url, page_section_generator=None):
     match = page_section_generator['regex'].search(url)
     return match.groupdict()
 
+def clean_invalid_args(url_args):
```

Comment:
Be sure to document the behavior of this method and what it returns

Reply:
I am not sure if it is the right way to document it, but I put down behaviour, args and return using standard docstrings.